### PR TITLE
Add fields necessary for Lendio integration

### PR DESCRIPTION
**Pull request by zalary**

### CODE REVIEW

**Comment by alexkwolfe** [comment]
Does the `company` field need to be deprecated?

**Comment by zalary** [comment]
Maybe?  Or does it make sense to keep the company field as the company name?  Maybe that goes against the way other dot notated have been set up

**Comment by alexkwolfe** [comment]
Yeah. I guess we could remove `company.name` from this PR and just use `company`, if you wanted.

**Comment by zalary** [comment]
I don't know, it's kind of weird either way.  Currently 'company' is under employment and has the description "Name of employer"  which seems irrelevant in the context of other integrations, and especially if company becomes a higher level field.  Maybe that description is okay to change and won't throw anyone off?  Or maybe company is a poor substitute for employer_name in the integrations where it exists? (primarily buyerzone but also hubspot, mortech and velocify)

**Comment by alexkwolfe** [comment]
I'm not sure we can answer that question right now, since we don't have a case of someone having a different company name and employer name. I suppose it could conceivably happen. Maybe we address it then?

**Comment by zalary** [comment]
Yea, that's fair.  I guess the descriptions aren't listed in the UI, so is changing the description ok?

**Comment by alexkwolfe** [comment]
Yeah.

**Comment by ros3bud** [comment]
Is this ok to be merged?

File: fields/lendio.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{ "id":"lendio.affiliate",            "name":"Lendio Affiliate Id",         "type":"number",   "description":"Lendio Affiliate Id" }
+{ "id":"lendio.marketplace",          "name":"Lendio Marketplace",          "type":"boolean",  "description":"Internal Lendio marketplace field" }
+{ "id":"lendio.accounts_recievable",  "name":"Accounts Receivable Amount",  "type":"string",   "description":"Accounts Receivable amount" }
+{ "id":"lendio.monthly_charges",      "name":"Monthly Charges",             "type":"string",   "description":"Monthly charges via credit cards" }
```
Comment (alexkwolfe) [inline fields/lendio.json]: Should accounts_receiveable and monthly_charges be numbers?

File: fields/lendio.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{ "id":"lendio.affiliate",            "name":"Lendio Affiliate Id",         "type":"number",   "description":"Lendio Affiliate Id" }
+{ "id":"lendio.marketplace",          "name":"Lendio Marketplace",          "type":"boolean",  "description":"Internal Lendio marketplace field" }
```
Comment (alexkwolfe) [inline fields/lendio.json]: Consider removing the extraneous "field" moniker from the description.

File: fields/company.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{"id":"company",                  "name":"Company Name",              "type":"string",  "description":"Name of company/employer"}
+{"id":"company.address_1",        "name":"Company Address 1",         "type":"string",  "description":"Company Address 1"}
```
Comment (alexkwolfe) [inline fields/company.json]: Should the type be "address"?

File: fields/company.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+{"id":"company.city",             "name":"Company City",              "type":"string",  "description":"Company City"}
+{"id":"company.state",            "name":"Company State",             "type":"string",  "description":"Company State"}
+{"id":"company.postal_code",      "name":"Company Postal Code",       "type":"string",  "description":"Company Postal Code"}
```
Comment (alexkwolfe) [inline fields/company.json]: Should the above three fields be typed also?

File: fields/lendio.json (hunk 1 above)
Comment (zalary) [inline fields/lendio.json]: Lendio expects these as ranges with dollar signs and commas `$4,001 - $8,000` so we could make them ranges and then transform the data that goes into those fields on the outbound?

Comment (alexkwolfe) [inline fields/lendio.json]: Hmm. Well, I guess these fields are specific to Lendio. Maybe leaving them as strings is fine.

Comment (alexkwolfe) [inline fields/lendio.json]: It seems weird to have a lead vendor specify the Lendio affiliate ID and Lendio marketplace. Are these fields better suited as request variables on the Lendio integration?

Comment (alexkwolfe) [inline fields/lendio.json]: The `accounts_recievable` field is incorrectly spelled.

Comment (zalary) [inline fields/lendio.json]: i blame rubymine.  catch my typos :(

Comment (zalary) [inline fields/lendio.json]: for the affiliate id and marketplace, I agree it makes more sense to have them as request variables.  would I just remove them as fields and then they would show up as field mapping options if they were designated in the integration?

**Comment by ros3bud** [comment]
Can this be merged now?

**Comment by alexkwolfe** [comment]
LGTM

**Pull request merged**